TicTacToe/TicTacToe.cs

```diff
@@ -14,63 +14,102 @@ class Program
 
         public static void Main()
         {
+            int tieInt = 0;     //if reaches 9, game ends
             do
             {
+                int[] newArr = new int[2];
+                DrawBoard();
+                newArr = GetInput();
+                if(PlaceMark(newArr[0],newArr[1]) == true && playerTurn == "X")
```

Comment:
You are calling PlaceMark twice, once in the if and once in the else if. This could be  a problem.